recipe/meta.yaml

```diff
@@ -1,32 +1,39 @@
-{% set version = '0.7.0' %}
+{% set version = '0.7.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
 
 package:
   name: r-ritis
-  version: {{ version }}
+  version: {{ version|replace("-", "_") }}
 
 source:
   fn: ritis_{{ version }}.tar.gz
   url:
-    - https://cran.r-project.org/src/contrib/ritis_{{ version }}.tar.gz
-    - https://cran.r-project.org/src/contrib/Archive/ritis/ritis_{{ version }}.tar.gz
-  sha256: bb3a3243f83083e7dd874624e8dfb57953c8aea38df16b3e5c0163857e875d38
+    - {{ cran_mirror }}/src/contrib/ritis_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ritis/ritis_{{ version }}.tar.gz
+  sha256: 3a8c57fd498b6cbe7e4e2aa015b176d9d699b249f390735d0a5eb8f49eb3c7ae
 
 build:
+  merge_build_host: True  # [win]
   number: 0
-  script: R CMD INSTALL --build .
```

Comment:
This ~~is~~ was the build script.

Reply:
Figures, I just used the skeleton to update the recipe assuming that it had previously been using the standard build.sh and such.